contrib/guix/libexec/build.sh

```diff
@@ -369,6 +369,8 @@ mkdir -p "$DISTSRC"
         # has not been run before buildling, this file will be a stub
         cp "${DISTSRC}/share/examples/bitcoin.conf" "${DISTNAME}/"
 
+        cp -r "${DISTSRC}/share/rpcauth" "${DISTNAME}/share/"
```

Comment:
Install a `share\rpcauth` folder using Windows installer as well?

Reply:
Added a (untested) change for this.